scripts/update-android-docs.sh

```diff
@@ -145,7 +145,6 @@ function create_pull_request() {
 gh auth login --with-token < gh_token.txt
 
 # Generate docs, create branch and make PR with API documentation in the SDK repo.
-generate_docs
```

Comment:
can we remove associated function from bash script as well?

Reply:
:+1: